### ISSUE TITLE
Accessibility question for "Alert With No Icon"

### ISSUE THREAD

**Issue by DanielMiller-agile6** [uswds/uswds#4185]
**Is your feature request related to a problem? Please describe.**
Related to the "Alert With No Icon" example described here:
https://designsystem.digital.gov/components/alert/

Does this introduce an accessibility issue? Without the icon, the only thing indicating the "type" of the alert is color.  Otherwise it seems like the alert's content would need to clearly indicate the type of alert in another way (if the type is required for understanding).

**Describe the solution you'd like**
Explanation describing the use-case for the "Alert with no icon."  Updated usage/accessibility guidelines in documentation if WCAG criteria can be violated.

**Describe alternatives you've considered**
If this can cause a WCAG violation, should the "alert with no icon" variant be available in USWDS?

**Additional context**
This is more of a question than a feature request, so no changes may be necessary.  I'm mainly just looking for some guidance since this came up during our project work.

**Comment by mejiaj** [comment]
@DanielMiller-agile6 it would be accessible as long as the message is clear and you're using the appropriate aria-role.

https://designsystem.digital.gov/components/alert/#alert-guidance

**Comment by jAigret-Bix** [comment]
There are no actionable items. Issue will be closed.
